### DISCUSSION TITLE
glspVscodeConnector  doesn't give deselected items info

### DISCUSSION THREAD

**Discussion by badrinarayan-R83** [eclipse-glsp/glsp#915] [General]
glspVscodeConnector.onSelectionUpdate gives an empty array if an element is deselected but provides no other info of the deselected item. Is there anyway to get list of deslected ids ?

**Comment by tortmayr** [comment]
Hi @badrinarayan-R83,
thank you for reporting this issue. 
At first glance this sounds like a bug to me. We have to look into it and have created #920 to track this
